plugins/org.polarsys.capella.cybersecurity.model/src/org/polarsys/capella/cybersecurity/model/helpers/CybersecurityHelpers.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *   
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.cybersecurity.model.helpers;
+
+import java.util.Optional;
+import org.eclipse.sirius.business.api.session.Session;
+import org.eclipse.sirius.ui.business.api.session.SessionEditorInput;
+import org.eclipse.ui.IEditorInput;
+import org.eclipse.ui.IEditorPart;
+import org.eclipse.ui.PlatformUI;
+import org.polarsys.capella.core.data.capellacore.EnumerationPropertyType;
+import org.polarsys.capella.core.data.capellamodeller.Project;
+import org.polarsys.capella.cybersecurity.model.CybersecurityPkg;
+import org.polarsys.kitalpha.emde.model.ElementExtension;
+import org.polarsys.capella.core.sirius.ui.helper.SessionHelper;
+
+public class CybersecurityHelpers {
+  public static final String CYBERSECURITY_CFG_KEYWORD = "Cybersecurity Configuration";
+  public static final String CYBERSECURITY_CFG_SECURITY_CONFIDENTIALITY_KEYWORD = "Security.Confidentiality";
+  public static final String CYBERSECURITY_CFG_SECURITY_INTEGRITY_KEYWORD = "Security.Integrity";
+  public static final String CYBERSECURITY_CFG_SECURITY_TRACEABILITY_KEYWORD = "Security.Traceability";
+  public static final String CYBERSECURITY_CFG_SECURITY_AVAILABILITY_KEYWORD = "Security.Availability";
+  
+  public static Project getCurrentProject() {
+    IEditorPart activeEditor = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().getActiveEditor();
```

Comment:
Can you avoid use of a getCurrentProject using the PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().getActiveEditor(); 

In methods below, you shall retrieve the project using 


for getEnumerationPropertyType used into getConfidentialitySize, getXXXSize, you can retrieve the project by using the view parameter from caller method :  getSecurityNeedsGreen(view)

for getEnumerationPropertyType, used in createEnumerationLiteralValueRadioGroup, / createControl, you can retrive the project while the loadData (maybe create control in this if not yet created)

Reply:
Ok, I'll analyze.